starwars_api/models.py

```diff
@@ -11,15 +11,20 @@ def __init__(self, json_data):
         Dynamically assign all attributes in `json_data` as instance
         attributes of the Model.
         """
-        pass
-
+        self.data = json_data
+        
+        for keys,values in json_data.items():
+            setattr(self, keys, values)
+        
     @classmethod
     def get(cls, resource_id):
         """
         Returns an object of current Model requesting data to SWAPI using
         the api_client.
         """
-        pass
+        method = getattr(api_client,"get_"+cls.RESOURCE_NAME)
```

Comment:
It's a good convention to surround operators (`+`, `,`, etc) with white spaces:
`api_client, "get_" + cls.RESOURCE_NAME`